neutrona/datadog_checks/neutrona/data/conf.yaml.example

```diff
@@ -0,0 +1,12 @@
+init_config:
```

Comment:
Hello, in order to follow [the integrations configuration  file guidelines](https://github.com/DataDog/integrations-core/blob/master/docs/dev/new_check_howto.md#configuration-file) the configuration file should have this shape:

```
init_config:

instances:

## @param azure - dictionary - required
## Azure authentication parameters.
## Replace all placeholders below to set up your Azure Neutrona integration.
#
  - azure:

  ## @param directory_id - string - required
  ## Directory id of your Azure account.
  #
    directory_id: "<DIRECTORY_ID>"
  
  ## @param application_id - string - required
  ## Application id of your Azure account.
  # 
    application_id: "<APPLICATION_ID>"
  
  ## @param application_key - string - required
  ## Application Key of your Azure account.
  #   
    application_key: "<APPLICATION_KEY>"
  
  ## @param domain - string - required
  ## The Azure domain for your account.
  # 
    domain: "<AZURE_DOMAIN>"
  
  ## @param subscription_id - string - required
  ## Subscription id of your Azure account.
  # 
    subscription_id: "<SUBSCRIPTION_ID>"
```

Feel free to edit the parameter desc with link to relevant documentation of more precisions if need be.